packages/documentation-v7/src/stories/misc/changelog.stories.mdx

```diff
@@ -0,0 +1,16 @@
+import { Markdown, Meta } from '@storybook/blocks';
+import changelog from '../../../../styles/CHANGELOG.md?raw';
+
+<Meta
+  title="Misc/ChangeLog"
+  parameters={{
```

Comment:
These parameters can most likely be removed: there is no canvas in the page and the `viewMode` is no longer needed in storybook 7 since docs have been separated from stories.

Reply:
Thanks! Yes, I don't see any changes.